zfec/cmdline_zunfec.py

```diff
@@ -3,18 +3,24 @@
 # zfec -- a fast C implementation of Reed-Solomon erasure coding with
 # command-line, C, and Python interfaces
 
-import os, sys
+from __future__ import print_function
 
-import argparse
-import filefec
+import os, sys, argparse
+
+from zfec import filefec
 
 from zfec import __version__ as libversion
 __version__ = libversion
 
+try:
+    unicode = unicode
+except NameError:
+    unicode = str
```

Comment:
also, we shouldn't be afraid to use `six` for this sort of compatibility stuff, it's pretty small and used by a lot of other packages (including Tahoe)